src/disclosures/js/views/financial-view.js

```diff
@@ -619,6 +624,19 @@ var financialView = {
   },
 
   /**
+   * Sets visibility of graduation cohort content. Hidden if not available.
+   * @param {boolean} visibility - Whether or not a graduation cohort
+   * was provided
+   */
+  graduationCohortVisible: function( visibility ) {
```

Comment:
Instead of `graduationCohortVisible` I'd suggest making the name `setGraduationCohortVisibility`, since while a bit longer, that describes better what the method is doing.